src/components/MessageBar/MessageBar.jsx

```diff
@@ -10,35 +19,138 @@ import {
   SendIcon,
 } from './MessageBarStyled';
 
-const MessageBar = () => {
-  const [value, setValue] = useState('');
+const MessageBar = ({
+  countryData,
+  setCountryData,
+  setSubscriptionCountryRooms,
+}) => {
+  const [message, setMessage] = useState('');
+  const [isShowJoinBtn, setIsShowJoinBtn] = useState(true);
+  const userId = useSelector(getUser)?.id;
   const textAreaRef = useRef(null);
+  const stompClient = useStompClient();
```

Comment:
You have stompClient in useWebSocket hook.

Reply:
Done